brianlong/www.validators.app
# 187930491 Upgrade node and webpacker

#### What's this PR do?
- Upgrade node to v18 and webpacker to latest 5.x versions, and all necessary js packages

#### How should this be manually tested?
- remove old node modules and old compiled assets
- make sure you use correct node version and run `yarn install` and then `rails assets:precompile`
- run the server and click through the app, especially pages with vue components (eg. ping thing, validator details), confirm that it works as expected

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/187930491)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
